SETUP.md

```diff
@@ -143,9 +148,9 @@ These steps are for OSX devices, including Apple Macbooks running on [Apple Sili
     1. Run `brew install openssl`
     2. Following the instructions in the output, run a form of `export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/`
 
-1. Install [Google Chrome](https://www.google.com/chrome/), needed for some local app tests.
-
-1. For Apple Silicon (M1), make these changes to your [Gemfile.lock](Gemfile.lock) file to switch from `libv8` to `libv8-node` and upgrade `mini_racer`. These changes should not be committed and will unfortunately clutter your `git status`.
+1. Install [Google Chrome](https://www.google.com/chrome/), needed for some local app tests. 
+  
+1. Return to the [Overview](#overview) to continue installation and clone the code-dot-org repo. Afer cloning, you will need to make the changes below to your [Gemfile.lock](Gemfile.lock) file to switch from `libv8` to `libv8-node` and upgrade `mini_racer`. These changes should not be committed and will unfortunately clutter your `git status`.
```

Comment:
So glad you added this part!

Reply:
I believe this instruction is specific to M1 macs; I think it's worth keeping the `For Apple Silicon (M1),` bit